antlr_tsql/ast.py

```diff
@@ -324,11 +330,28 @@ class OrderByExpr(AliasNode):
     _rules = ["order_by_clause"]
 
 
+class GroupByGroupingSets(AliasNode):
+    _fields_spec = ["grouping_set"]
+    _rules = ["group_by_grouping_sets"]
+
+
 class SortBy(AliasNode):
     _fields_spec = ["expr=expression", "direction"]
     _rules = ["order_by_expression"]
 
 
+class TimeZoneConversion(AliasNode):
+    _fields_spec = ["expr=left", "timezone=right"]
+    _rules = ["conversion_expression"]
+
+
+"""
+class ExpressionArgument(AliasNode):
```

Comment:
What will this be used for in the future?

Reply:
It was a previous class that I wanted to create, but I forgot to delete the comment. Fixed.